docs/sdk/client-side-sdks/react-native.md

```diff
@@ -53,56 +52,45 @@ Example of the above steps:
 import React from 'react'
 import 'react-native-get-random-values'
 import DeviceInfo from 'react-native-device-info'
-import { withDVCProvider } from '@devcycle/devcycle-react-sdk'
+import { withDVCProvider } from '@devcycle/devcycle-react-native-sdk'
 
 global.DeviceInfo = DeviceInfo
 ```
 
 5. Wrap your application component tree in either the `withDVCProvider` or `asyncWithDVCProvider` higher-order component (HOC), as explained in the [Getting Started](#getting-started) section.
 
-Pass in the option `reactNative: true` to the HOC to tell the SDK to run in React Native mode.
 
 ```jsx
-export default withDVCProvider(
-{
-	envKey: 'ENV_KEY',
-	options: {
-		reactNative: true
-	}
-})(App)
+export default withDVCProvider({ envKey: 'CLIENT_CLIENT_ENV_KEY' })(App)
 ```
 
 A complete working example of an `App.jsx` file is below:
 ```jsx
-import React from 'react';
-import { View, Text } from "react-native";
-
+import React from 'react'
+import { View, Text } from 'react-native'
 
 import 'react-native-get-random-values'
 import DeviceInfo from 'react-native-device-info'
-import { withDVCProvider } from '@devcycle/devcycle-react-sdk'
+import { withDVCProvider } from '@devcycle/devcycle-react-native-sdk'
 
 global.DeviceInfo = DeviceInfo
 function App() {
   return (
     <View
       style={{
         flex: 1,
-        justifyContent: "center",
-        alignItems: "center",
+        justifyContent: 'center',
+        alignItems: 'center',
       }}
     >
       <Text>Universal React with Expo</Text>
     </View>
-  );
+  )
 }
 
 export default withDVCProvider(
 {
-    envKey: 'client-xxxxxxxxxxxxxxxxxxx',
-    options: {
-        reactNative: true
-    }
+    envKey: 'client-xxxxxxxxxxxxxxxxxxx'
```

Comment:
should this use the same dummy key as the other examples?

Reply:
will fix! :) thanks